pkg/dogstatsd/listeners/uds_common_test.go

```diff
@@ -9,29 +9,37 @@
 package listeners
 
 import (
+	"github.com/stretchr/testify/require"
 	"io/ioutil"
 	"net"
 	"os"
 	"path/filepath"
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
```

Comment:
This order was actually good, we group by native/vendor/project dependencies

Reply:
Damn, GoLand took care of this for me. Maybe I'm missing some option to have the import grouped according this order.
Sorry about the noise.